README.md

```diff
@@ -577,7 +577,7 @@ Experimental Memcache support has been added as an alternative to Redis in v1.5.
 
 To configure a Memcache instance use the following environment variables instead of the Redis variables:
 
-1. `MEMCACHE_HOST_PORT=<host:port>`
+1. `MEMCACHE_URL=`: a comma separated list of hostname:port pairs for memcache nodes.
```

Comment:
I wasn't sure exactly how this behaves in the client library. Seems like the default behavior if you use a list of servers and not a custom selector is at https://github.com/bradfitz/gomemcache/blob/a41fca850d0b/memcache/selector.go#L114: it calculates ChecksumIEEE for a particular key has a hash into the list of backends.

Might be worth noting somewhere here in the documentation that when using a list of memcache notes the same list should be used on all rate limiters because backend selection is done based on the list and the key checksum.

Reply:
Right, same key should be hashed to same memcache node. That's a great point, will mention in docs.